src/js/views/admin/shared/clinicians_components.js

```diff
@@ -63,6 +66,122 @@ const RoleComponent = Droplist.extend({
   },
 });
 
+const GroupsItemView = View.extend({
+  tagName: 'li',
+  className: 'clinician-groups__item',
+  template: hbs`<span class="clinician-groups__item-icon">{{far "users"}}</span>{{name}}<button class="button--icon clinician-groups__remove js-remove">{{far "trash-alt"}}</button>`,
+  triggers: {
+    'click .js-remove': 'remove',
+  },
+});
+
+const ClinicianGroupsView = CollectionView.extend({
+  tagName: 'ul',
+  className: 'clinician-groups',
+  childView: GroupsItemView,
+  childViewOptions() {
+    return {
+      disabled: this.getOption('state').isDisabled,
+    };
+  },
+  childViewTriggers: {
+    'remove': 'remove',
+  },
+  onRemove(child) {
+    this.triggerMethod('remove:clinician', child.model);
+  },
+});
+
+const GroupsDropList = Droplist.extend({
+  initialize() {
+    this.listenTo(this.getOption('clinicianGroups'), 'add remove', this.show);
+  },
+  viewOptions() {
+    const clinicianGroups = this.getOption('clinicianGroups');
+
+    return {
+      className: 'button-secondary clinician-groups__droplist',
+      template: hbs`{{far "users"}}{{text}}`,
+      templateContext() {
+        return {
+          // TODO: i18n-ify
+          text: clinicianGroups.length > 0 ? 'Add Group...' : 'Select Group...',
+        };
+      },
+    };
+  },
+  picklistOptions: {
+    attr: 'name',
+    itemTemplate: hbs`<a>{{matchText text query}}</a>`,
+  },
+  onChangeSelected(selected) {
```

Comment:
elsewhere we're overriding `onPicklistSelect` to prevent state selection.. which should also potentially mean you don't need to override `itemTemplate` above.  Also is there a `!selected` scenario?